exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -127,19 +187,33 @@ void createTestKitWithBuilderForMultipleDifferentServicesVarargs() {
   void createTestKitWithTimeService() {
     TimeProvider timeProvider = FakeTimeProvider.create(TIME);
     try (TestKit testKit = TestKit.builder()
-        .withService(TestServiceModule.class)
-        .withTimeService(timeProvider)
+        .withDeployedArtifact(ARTIFACT_ID, ARTIFACT_FILENAME)
+        .withService(ARTIFACT_ID, SERVICE_NAME, SERVICE_ID)
+        .withTimeService(TIME_SERVICE_NAME, TIME_SERVICE_ID, timeProvider)
         .build()) {
-      TestService service = testKit.getService(TestService.SERVICE_ID, TestService.class);
+      TestService service = testKit.getService(SERVICE_NAME, TestService.class);
       checkTestServiceInitialization(testKit, service);
```

Comment:
That seems redundant and unrelated: why concern with services when we test time service deployment? We need to test that there is a time service with the given id and name.

Reply:
Initially this test was intended to test that if we create a TestKit with a time service, then user service is created as expected. We have different tests for time service - https://github.com/exonum/exonum-java-binding/pull/1145/files/afa48a202dc80d1fde5a3f5e515747843ae12899#diff-510e4cbda0fc830f9084e79a6d1f19e1R549

Reply:
I don't see a need to test _just_ that a user service is created, but that a time service with the given parameters is started. The linked test will work once it is migrated to the new time schema interface requiring a time service name, but we still need to test that multiple time services can be created — and the way the linked test is implemented seems overcomplicated for that purpose.